modules/tinymce/src/core/test/ts/browser/EditorUploadTest.ts

```diff
@@ -429,6 +429,22 @@ describe('browser.tinymce.core.EditorUploadTest', () => {
     return editor.uploadImages().then(() => assertEventsLength(1));
   });
 
+  it('TINY-9696: Removing an image should not leave the containing block without bogus element.', async () => {
+    const editor = hook.editor();
+    setInitialContent(editor, `<p>A</p><p></p><p>${imageHtml(testBlobDataUri)}</p>`);
+
+    editor.options.set('images_upload_handler', () => {
```

Comment:
Are we sure setting this doesn't effect other tests after this is they don't set this. So we don't get a pivot flake here.

Reply:
Both the previous and subsequent tests set this value on their own, so it should not cause any more issues that wouldn't already be caused.